app/src/main/res/layout/sheet_contact_patient.xml

```diff
@@ -6,13 +6,24 @@
   android:layout_height="wrap_content"
   android:layout_gravity="bottom">
 
+  <com.google.android.material.progressindicator.CircularProgressIndicator
+    android:id="@+id/progressIndicator"
+    android:layout_width="wrap_content"
+    android:layout_height="wrap_content"
+    android:layout_gravity="center"
+    android:indeterminate="true"
+    android:visibility="gone"
+    tools:indeterminate="false"
+    tools:progress="100"
+    tools:visibility="visible" />
+
   <org.simple.clinic.contactpatient.views.CallPatientView
     android:id="@+id/callPatientView"
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
     android:layout_gravity="bottom"
-    android:visibility="gone"
-    tools:visibility="visible" />
+    android:visibility="invisible"
+    tools:visibility="gone" />
```

Comment:
Can we update the visibility to show visible in tools because the whole point to adding tools visibility is for the preview.

Reply:
Oh, I meant to remove that entirely. We are showing a different view in the `tools` here. We are showing progress indicator in tools now. I must have forgotten to remove this bit. Will push this change.

Reply:
Done, pushed the changes ✅